### PR TITLE
[CAPPL-98] fix wrong standard capability should not block initialisation

### PR DESCRIPTION
**Pull request by agparadiso**
### Description
This pr introduces a fix to the [CAPPL-98](https://smartcontract-it.atlassian.net/browse/CAPPL-98) error in which whenever a standardcapability job spec had a wrong path to the LOOP binary, it was preventing the NOP UI to load.

### How was this tested
The error was reproduced locally by spinning up a node and introducing an invalid job spec & re-starting the node which would cause the NOP UI not to load. After the fix the same steps where reproduced validating the NOP UI was loading.

### Supports

[CAPPL-98]: https://smartcontract-it.atlassian.net/browse/CAPPL-98?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: core/services/standardcapabilities/standard_capabilities.go [hunk 1]
```diff
@@ -74,27 +79,36 @@ func (s *standardCapabilities) Start(ctx context.Context) error {
...
+		go func() {
+			defer s.wg.Done()
+			cctx, cancel := s.stopChan.NewCtx()
+			defer cancel()
 
-		if err = s.capabilitiesLoop.Service.Initialise(ctx, s.spec.Config, s.telemetryService, s.store, s.CapabilitiesRegistry, s.errorLog,
-			s.pipelineRunner, s.relayerSet, s.oracleFactory); err != nil {
-			return fmt.Errorf("error initialising standard capabilities service: %v", err)
-		}
+			if err = s.capabilitiesLoop.WaitCtx(cctx); err != nil {
```
Comment (bolekk) [inline core/services/standardcapabilities/standard_capabilities.go:95]: So what is going to happen now if the LOOPP can't be started? Are we going to hang here forever?

File: core/services/standardcapabilities/standard_capabilities.go [hunk 2]
```diff
@@ -74,27 +79,36 @@ func (s *standardCapabilities) Start(ctx context.Context) error {
...
 
-		s.log.Info("Started standard capabilities for job spec", "spec", s.spec, "capabilities", capabilityInfos)
+			s.log.Info("Started standard capabilities for job spec", "spec", s.spec, "capabilities", capabilityInfos)
+		}()
 
 		return nil
 	})
 }
 
 func (s *standardCapabilities) Close() error {
```
Comment (bolekk) [inline core/services/standardcapabilities/standard_capabilities.go:119]: Can we add a unit test with invalid config that calls Start() + Close()?

File: core/services/standardcapabilities/standard_capabilities.go [hunk 3]
```diff
@@ -74,27 +79,36 @@ func (s *standardCapabilities) Start(ctx context.Context) error {
 			return fmt.Errorf("error starting standard capabilities service: %v", err)
 		}
 
-		if err = s.capabilitiesLoop.WaitCtx(ctx); err != nil {
-			return fmt.Errorf("error waiting for standard capabilities service to start: %v", err)
-		}
+		s.wg.Add(1)
+		go func() {
+			defer s.wg.Done()
+			cctx, cancel := s.stopChan.NewCtx()
```
Comment (cedric-cordenier) [inline core/services/standardcapabilities/standard_capabilities.go]: @agparadiso Can we use the CtxWithTimeout variant of this method here and enforce some kind of timeout? Eg. a few minutes should be more than enough.

File: core/services/standardcapabilities/standard_capabilities.go (hunk 1 above)
Comment (bolekk) [reply core/services/standardcapabilities/standard_capabilities.go:95]: (where forever == 3 minutes)

**Review by justinkaseman — DISMISSED** [review]

**Review by bolekk — DISMISSED** [review]

**Review by justinkaseman — APPROVED** [review]

**Review by bolekk — APPROVED** [review]

**Pull request merged**